ILSpy.Tests/ILSpy.Tests.csproj

```diff
@@ -49,7 +49,7 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.7.0-2.final" />
     <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic" Version="3.7.0-2.final" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" />
-    <PackageReference Include="System.Collections.Immutable" Version="1.5.0" />
+    <PackageReference Include="System.Collections.Immutable" Version="1.7.1" />
```

Comment:
We should not be changing these as part of this change? I think they should already be in master.
The commits are getting convoluted, can we squash them?